### DISCUSSION TITLE
Remount App component

### DISCUSSION THREAD

**Discussion by solncebro** [vercel/next.js#44344] [Help]
Hi, everyone!
Do you know, why when I starting nextjs app, App.tsx mount, then unmount and then mount again?
I though something wrong with my application, but I created sandbox nextjs app and saw the same problem.

https://codesandbox.io/p/sandbox/infallible-greider-h0tkl8

![image](https://user-images.githubusercontent.com/42976880/209478223-22ede472-d69d-4b45-b2d6-4530b83d9803.png)

**Comment by wwwdepot** [comment]
It's React 18 strict mode (dev only, you won't see it in prod), turn it off, this behavior will go away.
